NuimoSDK/NuimoBluetoothController.cs

```diff
@@ -153,12 +153,12 @@ private bool ReadCharacteristicValue(Guid characteristicGuid, Action<byte[]> onV
             return !readResult.IsCanceled;
         }
 
-        public async void DisplayLedMatrixAsync(NuimoLedMatrix matrix, double displayInterval = 2.0, int options = 0)
+        public async void DisplayLedMatrixAsync(NuimoLedMatrix matrix, double displayInterval = 2.0, NuimoLedMatrixWriteOption options = NuimoLedMatrixWriteOption.None)
         {
             if (ConnectionState != NuimoConnectionState.Connected) return;
 
-            var withFadeTransition   = (options & (int) NuimoLedMatrixWriteOption.WithFadeTransition)   != 0;
-            var writeWithoutResponse = (options & (int) NuimoLedMatrixWriteOption.WithoutWriteResponse) != 0;
+            var withFadeTransition   = options.HasFlag(NuimoLedMatrixWriteOption.WithFadeTransition);
+            var writeWithoutResponse = options.HasFlag(NuimoLedMatrixWriteOption.WithoutWriteResponse);
```

Comment:
Now, that flag testing is cleaner, can you please get rid of the variables `withFadeTransition` and `writeWithoutResponse` and inline their values?

E.g.
- `options.HasFlag(NuimoLedMatrixWriteOption.WithoutWriteResponse)` in line 169
- `byteArray[10] |= Convert.ToByte(options.HasFlag(NuimoLedMatrixWriteOption.WithFadeTransition) ? 1 << 4 : 0);` in line 165

Thanks for this nice PR!